### PR TITLE
Add db seeds

### LINKED ISSUE #5: Image Seeds
#### As a developer I want at least 20 images to be created when I setup the database.

_Reviewer: @bboe_

Acceptance criteria:
- [x] After running `rake db:setup` at least 20 images are shown in the
  homepage.
- [x] When a new app is deployed on heroku (e.g., every review app), that app
  has at least 20 images on its homepage.

Dependencies:
- Image Index

### PR DESCRIPTION
**Pull request by conproctor**
Closes #5 

- [x] I assigned the designated reviewer to this PR

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,14 @@
...
+Image.create! [{url: "http://i.imgur.com/vYuYhRv.gif"}, {url: "http://i.imgur.com/mJRa4GD.gif"}, \
+               {url: "http://i.imgur.com/yyky7Ox.gif"}, {url: "http://i.imgur.com/8dx98kd.gif"}, \
+               {url: "http://i.imgur.com/3CpHV0k.gif"}, {url: "http://i.imgur.com/Z3rtFAT.gif"}, \
+               {url: "http://i.imgur.com/3qV3pPZ.gif"}, {url: "http://i.imgur.com/V4va1Sy.gif"}, \
+               {url: "http://i.imgur.com/cOoXLpI.gif"}, {url: "http://i.imgur.com/rN6IVrA.gif"}, \
+               {url: "http://i.imgur.com/tYV1OoV.gif"}, {url: "http://i.imgur.com/fQCsKr0.gif"}, \
+               {url: "http://i.imgur.com/c733DNU.gif"}, {url: "http://i.imgur.com/U5vKX2u.gif"}, \
+               {url: "http://i.imgur.com/zp6coEG.gif"}, {url: "http://i.imgur.com/mJRa4GD.gif"}, \
+               {url: "http://i.imgur.com/5yS3LEE.gif"}, {url: "http://i.imgur.com/PNeYj5X.gif"}, \
+               {url: "http://i.imgur.com/YZapNxk.gif"}, {url: "http://i.imgur.com/D6XS1k6.gif"}]
```
Comment (rmacklin) [inline db/seeds.rb]: Minor, but you could reduce some duplication here via:
```ruby
image_urls = %w(
  http://i.imgur.com/vYuYhRv.gif
  http://i.imgur.com/mJRa4GD.gif
  http://i.imgur.com/yyky7Ox.gif
  http://i.imgur.com/8dx98kd.gif
  http://i.imgur.com/3CpHV0k.gif
  http://i.imgur.com/Z3rtFAT.gif
  http://i.imgur.com/3qV3pPZ.gif
  http://i.imgur.com/V4va1Sy.gif
  http://i.imgur.com/cOoXLpI.gif
  http://i.imgur.com/rN6IVrA.gif
  http://i.imgur.com/tYV1OoV.gif
  http://i.imgur.com/fQCsKr0.gif
  http://i.imgur.com/c733DNU.gif
  http://i.imgur.com/U5vKX2u.gif
  http://i.imgur.com/zp6coEG.gif
  http://i.imgur.com/mJRa4GD.gif
  http://i.imgur.com/5yS3LEE.gif
  http://i.imgur.com/PNeYj5X.gif
  http://i.imgur.com/YZapNxk.gif
  http://i.imgur.com/D6XS1k6.gif
)
Image.create!(image_urls.map { |url| { url: url } })
```

**Comment by rmacklin** [comment]
I like your images 😛

**Comment by conproctor** [comment]
Updated to use the suggested format for creating many images

**Comment by rmacklin** [comment]
Looks good!

**Pull request merged**